bullet_train/app/models/concerns/users/base.rb

```diff
@@ -2,6 +2,7 @@ module Users::Base
   extend ActiveSupport::Concern
 
   included do
+    extend Devise::Models
```

Comment:
Explicitly including this so that things will work in tests when the rail ties don't do the thing.